tee-worker/litentry/pallets/identity-management/src/identity_context.rs

```diff
@@ -38,18 +38,14 @@ pub struct IdentityContext<T: Config> {
 	// the sidechain block number at which the identity is linked
 	pub link_block: BlockNumberOf<T>,
 	// a list of web3 networks on which the identity should be used
-	pub web3networks: Vec<Web3Network>,
+	pub web3networks: Option<Vec<Web3Network>>,
```

Comment:
I think we need to remove it. We don't need to store it anymore. 
Just passed in as an option parameter when requesting VC. That would be enough.

Reply:
From what I understood, we should do the migration first and then remove it. Otherwise the migration will fail here https://github.com/litentry/litentry-parachain/pull/2736/files#diff-c45ca5a4e78440095e592f8784465f19278a7c2c23a0bedb9ed2a82d6817e233R67

Reply:
returning it back, since otherwise it can not read idgraph :(